docs/content/TextInputWithTokens.mdx

```diff
@@ -213,12 +244,19 @@ render(MaxHeightExample)
     description="Whether tokens should render inline horizontally. By default, tokens wrap to new lines"
   />
   <PropsTable.Row name="size" type="TokenSizeKeys" defaultValue="extralarge" description="The size of the tokens" />
+  <PropsTable.Row
+    name="inputSize"
+    type="'small' | 'large'"
+    defaultValue="'small'"
+    description="The size of the text input"
+  />
```

Comment:
This can be removed now with latest changes?

Reply:
Can we also update `TokenSizeKeys` to the actual string values like you did for `inputSize`? 
![Screenshot 2022-02-09 at 13 34 04](https://user-images.githubusercontent.com/13340707/153211697-48277361-129b-4280-b2e4-ced4f2e62e18.png)

Reply:
> Can we also update TokenSizeKeys to the actual string values like you did for inputSize?

I agree that makes more sense, but in other docs we use the type name. For example: https://primer.style/react/ActionList

Reply:
I talked with @colebemis about this, and we're ok with a little inconsistency until we fix this in the other docs. I've updated https://github.com/primer/react/issues/1701 with this task.

Reply:
Hmm, okay.. it's just expensive to defer something that can be easily resolved now to avoid a heavier context switching cost later. 

It also seems that we have existing precedents, [where we can link directly to the interface/type definitions](https://primer.style/react/Flash) like this:
![Screenshot 2022-02-10 at 13 33 57](https://user-images.githubusercontent.com/13340707/153418685-acb5e066-a705-49f9-98f6-e97fee7ec993.png)

Can we do the same thing here?

Reply:
Yup, I changed it to be the whole string union.